content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -107,89 +99,123 @@ Once you have a Linux-based Kubernetes control-plane node you are ready to choos
     kube-system   kube-flannel-ds-54954                     1/1          Running   0          1m
     ```
 
-1. Add Windows Flannel and kube-proxy DaemonSets
-
-    Now you can add Windows-compatible versions of Flannel and kube-proxy. In order
-    to ensure that you get a compatible version of kube-proxy, you'll need to substitute
-    the tag of the image. The following example shows usage for Kubernetes {{< param "fullversion" >}},
-    but you should adjust the version for your own deployment.
-
-    ```bash
-    curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/{{< param "fullversion" >}}/g' | kubectl apply -f -
-    kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
-    ```
-    {{< note >}}
-    If you're using host-gateway use https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-host-gw.yml instead
-    {{< /note >}}
-
-    {{< note >}}
-If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, you have to modify the line:
+### Joining a Windows worker node
 
-```powershell
-wins cli process run --path /k/flannel/setup.exe --args "--mode=overlay --interface=Ethernet"
-```
+{{< note >}}
+All code snippets in Windows sections are to be run in a PowerShell environment
+with elevated permissions (Administrator) on the Windows worker node unless otherwise noted.
+{{< /note >}}
 
-in the `flannel-host-gw.yml` or `flannel-overlay.yml` file and specify your interface accordingly.
+{{< tabs name="tab-windows-kubeadm-runtime-installation" >}}
 
-```bash
-# Example
-curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml | sed 's/Ethernet/Ethernet0 2/g' | kubectl apply -f -
-```
-    {{< /note >}}
+{{% tab name="CRI-containerD" %}}
 
+### Intro
```

Comment:
We never have a heading “introduction” in our documentation. Actual introductory text should appear before the first heading at `<h2>` level.

Please reword this heading. Alternatively, you can list prerequisites under the Prerequisites heading (line 20 ish).